server/data/quotes.json

```diff
@@ -1,4 +1,8 @@
 ﻿[
+   {
+    "sentence": "With languages,you are at home anywhere.",
```

Comment:
This is a duplicate of the quote in line 275. Please use another quote :)

Reply:
ok